esmvaltool/diag_scripts/cmug_h2o/diag_tropopause.py

```diff
@@ -0,0 +1,239 @@
+    elif variable == "Geopotential Height":
+        print_var = "Geopotential Height [m]"
+        set_range = np.linspace(10000, 20000, 21)
+    elif variable == "Relative Humidity":
+        print_var = "Relative Humidity [%]"
+        set_range = np.linspace(0, 100, 21)
+    elif variable == "Specific Humidity":
+        print_var = "Specific Humidity [kg/kg]"
+        set_range = np.linspace(0.1e-5, 2.5e-5, 25)
+    else:
+        print_var = mean_cube.long_name
+        set_range = np.linspace(np.nanmin(mean_cube.data),
+                                np.nanmax(mean_cube.data), 21)
+    print("mean_cube.units")
+    print(mean_cube.units)
+    # draw filled contours
+    cnplot = plt.contourf(
+        lon_c,
+        mean_cube.coord('latitude').points,
+        data_c,
+        set_range,
+        transform=cart.PlateCarree(),
+        cmap='rainbow',
+        # cmap='RdBu_r',
+        extend='both')
+
+    axx.coastlines()
+
+    # add colorbar
+    cbar = fig.colorbar(cnplot, ax=axx, shrink=0.8)
+    # add colorbar title string
+    cbar.set_label(print_var)
+
+    axx.set_xlabel('Longitude')
+    axx.set_ylabel('Latitude')
+    axx.set_title(titlestr + variable)
+    # axx.set_xticks([40, 65, 90, 115])
+    # axx.set_xticklabels(['40°E', '65°E', '90°E', '115°E'])
+    # axx.set_yticks([-10, 0, 10, 20, 30])
+    # axx.set_yticklabels(['10°S', '0°', '10°N', '20°N', '30°N'])
+
+    fig.tight_layout()
+    figname = 'fig_' + titlestr.replace(" ", "_") +\
+        variable.replace(" ", "_") + '_map'
+    fig.savefig(get_plot_filename(figname, cfg), dpi=300)
+    plt.close()
+
+#    titlestr = titlestr + ' Box displays the area used to define the ' + \
+#        'average ISM (Indian Summer Monsoon) rainfall. Precipitation ' + \
+#        'changes are normalized by the corresponding global ' + \
+#        'mean SST increase for each model.'
+#
+#    selection = _get_sel_files_var(cfg, ['pr', 'ts'])
+#
+#    provenance_record = get_provenance_record(selection,
+#                                              titlestr, ['diff'], ['reg'])
+#
+#    diagnostic_file = get_diagnostic_filename(figname, cfg)
+#
+#    logger.info("Saving analysis results to %s", diagnostic_file)
+#
+#    iris.save(cube_to_save_ploted(data, lats, lons, {'var_name': 'd_pr',
+#                                                     'long_name': 'Prec' +
+#                                                                  'ipita' +
+#                                                                  'tion ' +
+#                                                                  'Change',
+#                                                     'units': 'mm d-1'}),
+#              target=diagnostic_file)
+#
+#    logger.info("Recording provenance of %s:\n%s", diagnostic_file,
+#                pformat(provenance_record))
+#    with ProvenanceLogger(cfg) as provenance_logger:
+#        provenance_logger.log(diagnostic_file, provenance_record)
+
```

Comment:
can you please remove commented out code or reinstate it if needed?

Reply:
hm, seems like I never completed the provenance here, trying to do that now

Reply:
Provenance added for esmvaltool/diag_scripts/cmug_h2o/diag_tropopause.py, but it is also missing for a part of the figures in esmvaltool/diag_scripts/cmug_h2o/diag_tropopausezonal.py

Reply:
@katjaweigel is this now ready for a final look before merging this into main?

Reply:
Sorry, on more figure type in esmvaltool/diag_scripts/cmug_h2o/diag_tropopausezonal.py missing, I hope I'll manage now

Reply:
@axel-lauer it should be complete now. I'm running an other tests for all parts on Mistral now (so far I only ran the diagnostics I changed, not deangelis).